### PR TITLE
[FCS] Implement Setter/Getter runtime variables

### PR DESCRIPTION
**Pull request by Abhishk123**

### CODE REVIEW

File: cypress/support/cypress-support/src/utils.js [hunk 1]
```diff
@@ -741,6 +741,72 @@ class FireLog {
...
+ * @description Resolve the device variable from the preprocessed data for the given key
+ * @example
+ * resolveAtRuntime(["result.{{attribute}}", "result.styles.{{attribute}}"])
+ * resolveAtRuntime("manage_closedcaptions.set{{attribute.uppercaseFirstChar}}")
+ * resolveAtRuntime("value")
+ *
+ * @returns
+ * ['result.fontSize', 'result.styles.fontSize']
+ * "manage_closedcaptions.setFontSize"
+ * "1.5"
```
Comment (jbigel) [inline cypress/support/cypress-support/src/utils.js:756]: I think the code probably handles this just fine but if the value is set to `1.5`, it should return as a number not as a String

File: cypress/support/cypress-support/src/utils.js [hunk 2]
```diff
@@ -741,6 +741,72 @@ class FireLog {
...
+    else if (!input.includes('{{')) {
+      return getEnvVariable(runtime).input || input;
+    }
+  } else if (Array.isArray(input) && input.length > 0) {
+    // input is an array; iterating through each element, it updates the actual value for that pattern if there is an occurrence of "{{".
+    return input.map((element) => {
+      if (element.includes('{{')) {
+        return replacingPatternOccurenceWithValue(element);
+      }
+    });
```
Comment (jbigel) [inline cypress/support/cypress-support/src/utils.js:804]: Can we add some log if input is neither a String or an Array.

**Review by jbigel — CHANGES_REQUESTED** [review]

File: cypress/support/cypress-support/src/utils.js (hunk 1 above)
Comment (Abhishk123) [reply cypress/support/cypress-support/src/utils.js:756]: it is returning the value present in rutime env as-is, this i missed to remove quote

File: cypress/support/cypress-support/src/utils.js (hunk 2 above)
Comment (Abhishk123) [reply cypress/support/cypress-support/src/utils.js:804]: addressed the comment

**Review by jbigel — APPROVED** [review]

**Review by neeradanelxsi — APPROVED** [review]

**Pull request merged**